settings/timesheets.sublime-settings

```diff
@@ -5,5 +5,19 @@
 
     // Jira ticket URL, e.g.
     // "https://jira.example.com/browse/{}"
-    "jira_ticket_url": "https://jira.iponweb.net/browse/{}"
-}
\ No newline at end of file
+    "jira_ticket_url": "https://jira.iponweb.net/browse/{}",
+
+    // JIRA endpoint that will be called to gather completion candidates
+    // Feel free to adjust the query and the number of results here.
+    "jira_completion_url": "https://jira.iponweb.net/rest/api/2/search?jql=(labels+in+(nbt,sft)+OR+watcher+in+({username})+OR+assignee+in+({username})+OR+reporter+in+({username}))+ORDER+BY+updated+DESC,+created+DESC&maxResults=100&fields=id,key,summary",
```

Comment:
This is default settings file, I suggest to remove command-specific labels. So by default it'll work for any employee. And use "was" instead of "is".